website/docs/features/templates.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Templates
+---
+
+Taqueria plugins are able to define templates that can be used to generate project components such as contracts, tests, or operations. Templates are exposed via plugins and use the `create` task to create and register the templates output
+
+
+## Available Templates
+
+| Template           | Template  Name    | Plugin                   | Command                        | Description                                                        |
+|--------------------|-------------------|--------------------------|--------------------------------|--------------------------------------------------------------------|
+| Ligo contract      | `contract`   | `@taqueria\plugin-ligo`       | `taq create contract <path>`   | Instantiates a new Ligo contract at the provided `<path>` and registers the contract in the contract registry |
+| Archetype contract | `contract`   | `@taqueria\plugin-archetype`  | `taq create contract <path>`   | Instantiates a new Archetype contract at the provided `<path>` and registers the contract in the contract registry  |
```

Comment:
The archetype plugin has the command `create archetypeContract <path>`. There was some sort of name collision that was happening so that was the decision made

Reply:
Ah, thanks for the info @alexzbusko!